src/components/calcite-checkbox/calcite-checkbox.scss

```diff
@@ -24,8 +24,6 @@
   position: relative;
   user-select: none;
   -webkit-tap-highlight-color: transparent;
-  width: var(--calcite-checkbox-size);
-  height: var(--calcite-checkbox-size);
```

Comment:
I had to remove these because they were constraining the dimensions of the checkbox when a label was present, resulting in unwanted line breaks.